app/models/partner.rb

```diff
@@ -112,7 +114,7 @@ class Partner < ApplicationRecord
 
   validate :three_or_less_category_tags
 
-  validate :partnership_admins_must_add_tag, on: %i[create]
+  validate :partnership_admins_must_add_partnership, on: %i[create]
```

Comment:
This is from the other PR I think. Maybe we close this and open a new one with just the intended changes... so we don't have a tangle later.

Reply:
Yeah all this is. I don't mind re-doing it if it's really needed, but won't this all get automatically fixed when main is updated? Keen to understand git a little better (and yes be a bit lazy if I can lol)